### PR TITLE
Fix BatchNormalization docstring

### LINKED ISSUE #2900: Types of BatchNormalization link are described inaccurately
Although `avg_mean` and `avg_var` are `ndarray`, the docstring says that they are `Variable`.

https://github.com/chainer/chainer/blob/6a526e53aee773b65dca7f0383890fa0ad9a9fac/chainer/links/normalization/batch_normalization.py#L56-L57

### PR DESCRIPTION
**Pull request by niboshi**
Fixes #2900

### CODE REVIEW

**Comment by mitmul** [comment]
LGTM
